client/src/components/ReportCollection.js

```diff
@@ -129,7 +168,13 @@ ReportCollection.propTypes = {
 }
 
 ReportCollection.defaultProps = {
-  viewFormats: [FORMAT_SUMMARY, FORMAT_TABLE, FORMAT_CALENDAR, FORMAT_MAP]
+  viewFormats: [
+    FORMAT_STATISTICS,
+    FORMAT_SUMMARY,
+    FORMAT_TABLE,
+    FORMAT_CALENDAR,
+    FORMAT_MAP
+  ]
```

Comment:
This changes the default tab from ReportSummary to ReportStatistics for all ReportCollections that don't explicitly specify viewFormats (user-facing change; should be in the PR description).